### PR TITLE
fixed bootstrap css interference

### PR DESCRIPTION
**Pull request by farazahmad759**

### CODE REVIEW

File: swiftplugin/swiftplugin/static/js/src/swiftplugin.js [hunk 1]
```diff
@@ -9,32 +9,26 @@ function SwiftPluginXBlock(runtime, element) {
...
+            data: JSON.stringify({ type: 'run', code: user_code, language: mode }),
             success: updateResponse,
             error: handleError
         });
     }
 
     const submit_btn = document.getElementById('submit-btn');
     submit_btn.onclick = function (eventObject) {
-        const {user_code, mode} = getCodeAndMode();
+        var user_code = myCodeMirror.getValue();
```
Comment (dri94) [inline swiftplugin/swiftplugin/static/js/src/swiftplugin.js:27]: why are these lines changed?

Comment (farazahmad759) [reply swiftplugin/swiftplugin/static/js/src/swiftplugin.js:27]: Please ignore them. I was experimenting with something, and later forgot to remove these lines.

Comment (dri94) [reply swiftplugin/swiftplugin/static/js/src/swiftplugin.js:27]: Okay, lets remove this line. PR looks good. Once these lines are removed I can give it a final test
